Dockerfile

```diff
@@ -180,15 +180,9 @@ RUN pip install scipy && \
     # PyTorch
     export CXXFLAGS="-std=c++11" && \
     export CFLAGS="-std=c99" && \
-    conda install -y pytorch torchvision cpuonly -c pytorch && \
-    # PyTorch Audio
     apt-get install -y sox libsox-dev libsox-fmt-all && \
```

Comment:
Nit: I think that we don't need to set complier flags or install sox and cffi anymore unless they are needed for something else.

Reply:
Thanks for flagging. I will try removing them in a separate PR.